src/main/resources/static/app/lab-request/edit-lab-request.html

```diff
@@ -220,11 +220,11 @@ <h4>{{'Lab request status'|translate}}: {{'Approved'|translate}}</h4>
                         <div class="panel-footer clearfix">
                             <span class="pull-right">
                                 <button class="btn btn-default btn-primary"
-                                        ng-if="labRequest.request.materialsRequest"
+                                        ng-if="Request.isMaterialsRequest(labRequest.request)"
```

Comment:
Does this work, is `Request` available in the template? Perhaps `$scope.Request = Request` should be added to the `RequestController.js`?

Reply:
Could also add `isMaterialRequest` to the scope.